feature-swap-impl/src/main/java/io/novafoundation/nova/feature_swap_impl/domain/slippage/SlippageRepository.kt

```diff
@@ -0,0 +1,45 @@
+package io.novafoundation.nova.feature_swap_impl.domain.slippage
+
+import io.novafoundation.nova.common.utils.Percent
+
+interface SlippageRepository {
+
+    fun getDefaultSlippage(): Percent
+
+    fun getSlippageTips(): List<Percent>
+
+    fun minSlippage(): Percent
+
+    fun maxSlippage(): Percent
+
+    fun smallSlippage(): Percent
```

Comment:
OK for now, but the target approach is to define slippage configuration inside `AssetExchange`:

``` interface AssetExchange {
...

suspend fun slippageConfig(): SlippageConfig
}
```

We can leave repository approach for now though